client/src/components/Chart.js

```diff
@@ -0,0 +1,37 @@
+import useLayout from "layouts/useLayout"
+import { LAYOUT_AGGREGATORS } from "layouts/utils"
+import PropTypes from "prop-types"
+import React, { useState } from "react"
+
+const Chart = ({ items, layoutType, widgetElement, widgetConfig, style }) => {
+  const aggregator = LAYOUT_AGGREGATORS[layoutType]
+  const [aggregatedItems] = aggregator(items)
```

Comment:
A chart should not assume it contains aggregations. It could be a simple bar chart.
It should take a collection of items, and map them to the appropriate GUI components based on how it is configured.
I suggest to have chart take a collection of items. If they have been aggregated, it should take as an input the list of aggregated groupings as items, with the original items referenced from the aggregations